src/linear_algebra/exterior_algebra.lean

```diff
@@ -115,7 +120,10 @@ begin
   refl,
 end
 
-attribute [irreducible] exterior_algebra ι lift
+attribute [irreducible] ι lift
+-- Marking `exterior_algebra` irreducible makes our `ring` instances inaccessible.
+-- https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/algebra.2Esemiring_to_ring.20breaks.20semimodule.20typeclass.20lookup/near/212580241
+-- For now, we avoid this by not marking it irreducible.
```

Comment:
Is this still accurate given the additions on line 69-71?

Reply:
Perhaps "inaccessible" is not quite right, but I think we'd still have problems unifying `ring_quot.ring` with `exterior_algebra.ring` if we marked `exterior_algebra` irreducible. At this point I'm just playing it safe - we know there are problems caused by `irreducible`, and it's easier just to be consistent.